temporal-spring-boot-autoconfigure-alpha/src/main/java/io/temporal/spring/boot/autoconfigure/template/WorkersTemplate.java

```diff
@@ -484,6 +501,7 @@ public List<RegisteredWorkflowInfo> getRegisteredWorkflowInfo() {
   public static class RegisteredActivityInfo {
```

Comment:
Can we mark these classes `RegisteredActivityInfo` and `RegisteredWorkflowInfo` as `@Experimental`? Sorry I should have suggested this in the last PR